### PR TITLE
Add ManageExtendedQueryTags data access role for administrate extended query tags

### PR DESCRIPTION
**Pull request by pengchen0692**
## Description
Add ManageExtendedQueryTags data access role for administrate extended query tags
## Related issues

Addresses [AB#81520](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/81520)

### CODE REVIEW

File: src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagHandler.cs [hunk 1]
```diff
@@ -30,9 +30,9 @@ public async Task<AddExtendedQueryTagResponse> Handle(AddExtendedQueryTagRequest
         {
             EnsureArg.IsNotNull(request, nameof(request));
 
-            if (await AuthorizationService.CheckAccess(DataActions.Write, cancellationToken) != DataActions.Write)
+            if (await AuthorizationService.CheckAccess(DataActions.Admin, cancellationToken) != DataActions.Admin)
```
Comment (smithago) [inline src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagHandler.cs:33]: Should we name it to be more specific? "ManageExtendedQueryTags"?

Comment (pengchen0692) [reply src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagHandler.cs:33]: Not sure about this,  other actions in DataActions https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Core/Features/Security/DataActions.cs are very generic (read/write/delete)

Comment (smithago) [reply src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagHandler.cs:33]: Admin feels overlapping to me. If you look at FHIR they had similar specific action for profiles I think

File: src/Microsoft.Health.Dicom.Core/Features/Security/DataActions.cs [hunk 2]
```diff
@@ -25,7 +25,10 @@ public enum DataActions : ulong
         [EnumMember(Value = "delete")]
         Delete = 1 << 2,
 
+        [EnumMember(Value = "manageExtendedQueryTag")]
```
Comment (snehabandla) [inline src/Microsoft.Health.Dicom.Core/Features/Security/DataActions.cs:28]: >manageExtendedQueryTag [](http://example.com/codeflow?start=29&length=22)

can we make this plural? 'manageExtendedQueryTags'

**Review by snehabandla — APPROVED** [review]
:shipit:

**Pull request merged**